images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -362,6 +362,15 @@ _ocp_enabled_overrides:
       config:
         prometheus:
           url: "https://prometheus-k8s.openshift-monitoring.svc:9091/"
+  presto:
+    spec:
+      config:
+        connectors:
+          prometheus:
+            enabled: true
+            uri: "https://prometheus-k8s.openshift-monitoring.svc:9091/"
```

Comment:
I think we may need to add a validation task that ensures this field isn't empty. That being said, we do validate the `reporting-operator.spec.config.prometheus.url` isn't empty when the ocp-only features are disabled.

Reply:
I think doing this only in the CRD was what we decided in Slack, right, so I am going to do that work on branch: `MC-720-crd-manifests-only`

Reply:
Yeah moving the validation from ansible-side to the CRD-side.